AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC/ALAppDelegate.m

```diff
@@ -19,6 +20,15 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     [ALSdk shared].mediationProvider = ALMediationProviderMAX;
     [[ALSdk shared] initializeSdkWithCompletionHandler:^(ALSdkConfiguration *configuration) {
         // AppLovin SDK is initialized, start loading ads now or later if ad gate is reached
+        
+        // initialize Adjust SDK
+        NSString *yourAppToken = @"{YourAppToken}";
+        NSString *environment = ADJEnvironmentSandbox;
+        ADJConfig *adjustConfig = [ADJConfig configWithAppToken:yourAppToken
+                                          environment:environment];
+
+        [adjustConfig setLogLevel:ADJLogLevelVerbose];
+        [Adjust appDidLaunch:adjustConfig];
```

Comment:
Formatting tips:

- Space after `:`,
- Good habit to get into is select all text via `⌘` + `A`, then `ctrl` + `I` to auto-indent, but our convention is not to break up < 3 parameters into new lines anyways.
```suggestion
        // Initialize Adjust SDK
        ADJConfig *adjustConfig = [ADJConfig configWithAppToken: @"{YourAppToken}" environment: ADJEnvironmentSandbox];
        [Adjust appDidLaunch: adjustConfig];
```